Update joins-and-nesting.mdx

slowlydeveloping opened the pull request:
I have also added the query where we can see the scan_id_end of the users, by their shifts.

## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

YES/NO

## What kind of change does this PR introduce?

Bug fix, feature, docs update, ...

## What is the current behavior?

Please link any relevant issues here.

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

slowlydeveloping committed:
Update joins-and-nesting.mdx

I have also added the query where we can see the scan_id_end of the users, by their shifts.

charislam commented:
Thanks @slowlydeveloped! Could you also update the text description before the code examples?

slowlydeveloping committed:
Update joins-and-nesting.mdx

I have updated the text description that perfectly summarizes what is being done below and also I have added the commands for kotlin and graphQl, which achieve the same outcome.

slowlydeveloped commented:
I have updated the text descriptions above the code and provided codes for kotlin and graphQL.

slowlydeveloping committed:
Merge branch 'master' into patch-2
